src/test/resources/alma/HT017664407.json

```diff
@@ -1,6 +1,6 @@
 {
-  "hbzId" : "HT017664407",
   "almaIdMMS" : "990197023370206441",
+  "hbzId" : "HT017664407",
```

Comment:
As discussed -if nothing of importance changes (here only the order changed, which can be ignored) we should not commit the resource.

Reply:
but this will be highlighted next time when we use this. also this shows that another marc element is used for the hbzId since it is "035" and not "009" which is used for the mapping.

as discussed off-board we keep these